wp-feed-post-thumbnail.php

```diff
@@ -26,18 +26,18 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
  */
 
-include( dirname( __FILE__ ) . '/lib/requirements-check.php' );
-
-$wp_feed_post_thumbnail_requirements_check = new WP_Feed_Post_Thumbnail_Requirements_Check( array(
-	'title' => 'WP Feed Post Thumbnail',
+if ( file_exists( dirname( __FILE__ ) . '/vendor/autoload.php' ) ) {
+	require dirname( __FILE__ ) . '/vendor/autoload.php';
+}
+$requirements_check = new WP_Requirements_Check( array(
```

Comment:
There's an `unset()` at the end of this file that also needs to be changed to the new variable name.